haddock-api/resources/html/NewOcean.std-theme/new-ocean.css

```diff
@@ -0,0 +1,896 @@
+  min-width: 9px;
+  min-height: 9px;
+}
+
+summary {
+  cursor: pointer;
+  outline: none;
+  list-style-image: url(plus.gif);
+  list-style-position: outside;
+}
+
+details[open] > summary {
+  list-style-image: url(minus.gif);
+}
+
+pre {
+  padding: 0.5rem 1rem;
+  margin: 1em 0 0 0;
+  background-color: #f7f7f7;
+  overflow: auto;
+}
+
+pre + p {
+  margin-top: 1em;
+}
+
+pre + pre {
+  margin-top: 0.5em;
+}
+
+.src {
+  background: #f4f4f4;
+  padding: 0.2em 0.5em;
+}
+
+.keyword { font-weight: normal; }
+.def { font-weight: bold; }
+
+@media print {
+  #footer { display: none; }
+}
+
+/* @end */
+
+/* @group Page Structure */
+
+#content {
+  margin: 3em auto 6em auto;
+  padding: 0;
+}
+
+#package-header {
+  background: #5E5184;
+  border-bottom: 5px solid rgba(69, 59, 97, 0.5);
+  color: #ddd;
+  position: relative;
+  font-size: 1.2em;
+  text-align: left;
+  margin: 0 auto;
+}
+
+#package-header .caption {
+  color: white;
+  font-style: normal;
+  font-size: 1rem;
+  font-weight: bold;
+}
+
+#module-header .caption {
+  font-weight: bold;
+  border-bottom: 1px solid #ddd;
+}
+
+table.info {
+  float: right;
+  padding: 0.5em 1em;
+  border: 1px solid #ddd;
+  color: rgb(78,98,114);
+  background-color: #fff;
+  max-width: 60%;
+  border-spacing: 0;
+  position: relative;
+  top: -0.78em;
+  margin: 0 0 0 2em;
+}
+
+.info th {
+  padding: 0 1em 0 0;
+  text-align: right;
+}
+
+div#style-menu-holder {
+  position: relative;
+  z-index: 2;
+  display: inline;
+}
+
+#style-menu {
+  position: absolute;
+  z-index: 1;
+  overflow: visible;
+  background: #374c5e;
+  margin: 0;
+  text-align: center;
+  right: 0;
+  padding: 0;
+  top: 1.25em;
+}
+
+#style-menu li {
+	display: inline-block;
+	border-style: none;
+	margin: 0;
+	padding: 0;
+	color: #000;
+	list-style-type: none;
+	border-top: 1px solid #919191
+}
+
+#style-menu li + li {
+	border-left: 1px solid #919191;
+}
+
+#style-menu a {
+  width: 6em;
+  padding: 3px;
+  display: block;
+}
+
+#footer {
+  background: #ededed;
+  border-top: 1px solid #aaa;
+  padding: 0.5em 0;
+  color: #222;
+  text-align: center;
+  width: 100%;
+  height: 3em;
+  margin-top: 3em;
+  position: relative;
+  clear: both;
+}
+
+/* @end */
+
+/* @group Front Matter */
+
+#synopsis .caption,
+#table-of-contents .caption {
+  font-size: 1rem;
+}
+
+#synopsis, #table-of-contents {
+  font-size: 16px;
+}
+
+#table-of-contents {
+  background:  #f7f7f7;
+  padding: 1em;
+  margin: 0;
+  margin-top: 1em;
+}
+
+#table-of-contents .caption {
+  text-align: left;
+  margin: 0;
+}
+
+#table-of-contents ul {
+  list-style: none;
+  margin: 0;
+  margin-top: 10px;
+  font-size: 14px;
+}
+
+#table-of-contents ul ul {
+  margin-left: 1.5em;
+}
+
+#description .caption {
+  display: none;
+}
+
+#synopsis summary {
+  display: block;
+  float: left;
+  width: 29px;
+  color: rgba(255,255,255,0);
+  height: 110px;
+  margin: 0;
+  font-size: 1px;
+  padding: 0;
+  background: url(synopsis.png) no-repeat 0px -8px;
```

Comment:
@NunoAlexandre  Doesn't `url(synopsis.png)` (and `url(plus.gif)` and `url(minus.gif)` elsewhere in this file) indicate you are still using these assets? Are we missing some of your newer changes? Forgive the potentially stupid question - I am not familiar at all with CSS...

Reply:
@harpocrates not a stupid question at all. Either some of my changes are missing indeed, or those styling elements referencing `synopsis` and `minus` were not taking any effect in the html pages I tested the style on and therefore I could have forgotten to delete them. They seem to be used though (considering your screenshots), therefore something's off. I will have a look 👍

Reply:
I did something with the synopsis before you commented your response. Feel free to revert that commit if you feel it clashes with everything else too much. I would love it if you took a look at the `minus` - there's one case that really feels like a bit of an eyesore:

<img width="154" alt="screen shot 2018-10-25 at 7 48 36 am" src="https://user-images.githubusercontent.com/10766081/47509240-88ca8580-d82a-11e8-9599-c4443a0bd1d6.png">

The old minus clashes with the new vertical indent line as well as with the new unicode pluses in the instances list. That said, aligning the `Instances` and `Constructors` section is obviously something we'd also like.